.vscode/cspell.json

```diff
@@ -2,6 +2,7 @@
   "version": "0.2",
   "language": "en",
   "useGitignore": true,
+  "allowCompoundWords": true,
```

Comment:
This prevents cspell errors from words like `azurecanary` in ci.yml that don't have case or other separators.

Reply:
Don't do that for *everything* then. use `# cspell:ignore <error>` above lines. We want to catch things elsewhere.